core/platform/search/elastic_search_services_test.py

```diff
@@ -217,9 +217,9 @@ def mock_search(body, index, params):
         with swap_search:
             result, new_offset = (
                 elastic_search_services.search(
-                    json_str, correct_index_name, offset=offset,
-                    limit=limit, ids_only=True))
-        self.assertEqual(new_offset, limit + offset)
+                    json_str, correct_index_name, cursor=offset,
```

Comment:
cursor=offset doesn't make sense.

Reply:
Changed back to original implementation with offset.